### PR TITLE
fix(kollider): routes

### LINKED ISSUE #2355: [BUG] Onboarding connectors: Kollider wallet Signup and Login buttons dont work
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

At the Kollider wallet screen where there is 2 buttons, clicking on one of them in the development build changes the location bar, but the screen remains.

[Kollider-routes.webm](https://user-images.githubusercontent.com/7760/230426042-fb9af585-bf32-4e8c-938f-1f21dc98073a.webm)

### Screenshots [optional]

image

### Steps To Reproduce

- clone the repo
- `yarn`
- `yarn dev:chrome`
- load unpacked extension

### Expected behavior

The screens change

### Alby information

- Alby commit b7cfb433b99d51a5e14137c8ed8f7979cfa4d94e

### Device information

I was able to reproduce it on the master branch for both Brave, Chrome and Firefox. With node versions `v16.18.0` and `v18.15.0`

### Additional context

There is a suggestion of fix [here](https://github.com/getAlby/lightning-browser-extension/pull/2309).

### Are you working on this?

No

### PR DESCRIPTION
**Pull request by fczuardi**
### Describe the changes you have made in this PR

Kollider wallet variants (buttons signup and login) were not working as expected.

### Link this PR to an issue [optional]

Fixes #2355

### Type of change

(Remove other not matching type)

- `fix`: Bug fix (non-breaking change which fixes an issue)
- `feat`: New feature (non-breaking change which adds functionality)
- `feat!`: Breaking change (fix or feature that would cause existing functionality to not work as expected)
- `docs`: Documentation update

### Screenshots of the changes [optional]

_Add screenshots to make your changes easier to understand. You can also add a video here._

### How has this been tested?

_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration_

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [ ] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Comment by im-adithya** [comment]
Thanks for the PR, but this change makes `/kollider` route empty

Also I am not able to reproduce the issue mentioned here: https://github.com/getAlby/lightning-browser-extension/issues/2250#issuecomment-1498394580

cc @reneaaron

**Comment by fczuardi** [comment]
> Thanks for the PR, but this change makes `/kollider` route empty
> 
> Also I am not able to reproduce the issue mentioned here: [#2250 (comment)](https://github.com/getAlby/lightning-browser-extension/issues/2250#issuecomment-1498394580)
> 
> cc @reneaaron

Here is a video displaying the steps to reproduce and the clicks:

[Kollider-routes.webm](https://user-images.githubusercontent.com/7760/230426042-fb9af585-bf32-4e8c-938f-1f21dc98073a.webm)

**Comment by fczuardi** [comment]
> makes `/kollider` route empty

nope, since there is an "index" child

File: src/app/router/Options/Options.tsx [hunk 1]
```diff
@@ -28,6 +28,41 @@ import ChooseConnector from "~/app/screens/connectors/ChooseConnector";
...
+        }[];
+      }[]
+    | {
+        index?: boolean;
+        element: JSX.Element;
+        path?: string;
+      }[]
+) {
+  return routes.map((route) => {
+    if ("children" in route && route.children) {
```
Comment (lujakob) [inline src/app/router/Options/Options.tsx:51]: Why not just `if (route?.children) {`

**Comment by lujakob** [comment]
@im-adithya could you please explain what the cause of the bug was and the reasoning behind fixing it. These changes will cause merge-conflicts with https://github.com/getAlby/lightning-browser-extension/pull/2272 that I'm working on.

File: src/app/router/Options/Options.tsx (hunk 1 above)
Comment (im-adithya) [reply src/app/router/Options/Options.tsx:51]: The `route` can also be a parent route for type guarding.

File: src/app/screens/connectors/ConnectKollider/index.tsx [hunk 2]
```diff
@@ -172,21 +172,7 @@ export default function ConnectKollidier({ variant }: Props) {
...
       description={
-        variant === "create" ? (
-          t(`create.description`)
-        ) : (
-          <Trans
-            i18nKey={"login.description"}
-            t={t}
-            components={[
-              // eslint-disable-next-line react/jsx-key
-              <Link
```
[the remark this answers is not in the document]
Comment (im-adithya) [reply src/app/screens/connectors/ConnectKollider/index.tsx:183]: Now that we have two routes serving the same component, we can't use the link... (`/welcome.html#/.../create` doesn't work for `options.html#/.../create`)

File: src/i18n/locales/en/translation.json [hunk 3]
```diff
@@ -291,7 +291,7 @@
         },
         "login": {
           "title": "Connect to your Kollider account",
-          "description": "Don't have an account already? <0>Sign up</0> now!"
+          "description": "Don't have an account already? Sign up now!"
```
Comment (reneaaron) [inline src/i18n/locales/en/translation.json:294]: @im-adithya Can you please update this PR like we discussed last week? This messages makes no sense here.

**Review by reneaaron — CHANGES_REQUESTED** [review]

File: src/i18n/locales/en/translation.json (hunk 3 above)
Comment (im-adithya) [reply src/i18n/locales/en/translation.json]: Fixed it to have links! Ready to merge now!

**Review by reneaaron — APPROVED** [review]
tACK

**Pull request merged**